presto-main/src/main/java/com/facebook/presto/sql/planner/iterative/rule/CanonicalizeExpressionRewriter.java

```diff
@@ -25,8 +25,12 @@
 import com.facebook.presto.sql.tree.QualifiedName;
 import com.google.common.collect.ImmutableList;
 
+import java.util.List;
+
 public class CanonicalizeExpressionRewriter
 {
+    private static final List<QualifiedName> rewriteEnabledFunctionNames = ImmutableList.of(QualifiedName.of("map_concat"), QualifiedName.of("array_concat"));
```

Comment:
Nits: static final variables usually have variable names in caps lock. Sorry for missing it previously.

Reply:
changed